src/import/sb3new.js

```diff
@@ -0,0 +1,16 @@
+var Sprite = require('../sprites/sprite');
+var Blocks = require('../engine/blocks');
+
+function sb3new (runtime) {

```

Comment:
sb3new is not really an "import" function, so I don't think it should live with the /import/ files. Rather it be a function of the VM's index.js, called e.g., `createNewProject`.